src/metabase/email/password_reset.mustache

```diff
@@ -5,11 +5,18 @@
       <a href="{{hostname}}">Go to {{applicationName}}</a>
     {{/sso}}
     {{^sso}}
-    <div style="text-align: center">
-      <p>Click the button below to reset the password for your {{applicationName}} account at {{hostname}}.</p>
-      <a style="display: inline-block; box-sizing: border-box; font-size: 18px; padding: 8px 22px; cursor: pointer; text-decoration: none; border-radius: 4px; background-color: #4990E2; border-color: #4990E2; color: #fff;" href="{{passwordResetUrl}}">Reset password</a>
-      <p style="padding-top: 2em; font-size: small;">Didn't request this password reset? It's safe to ignore it.</p>
-    </div>
+      {{#isActive}}
```

Comment:
is the formatting indented here?

Reply:
Not sure how mustache syntax usually works. I indented the `isActive` conditional block by a level since it's nested in the `sso` conditional. It doesn't affect the actual email formatting.